tests/mysql-types-time.rs

```diff
@@ -0,0 +1,92 @@
+use sqlx::types::time::{OffsetDateTime, Date, Time, UtcOffset};
+use sqlx::{mysql::MySqlConnection, Connection, Row};
+
+async fn connect() -> anyhow::Result<MySqlConnection> {
+    Ok(MySqlConnection::open(dotenv::var("DATABASE_URL")?).await?)
+}
+
+#[cfg_attr(feature = "runtime-async-std", async_std::test)]
+#[cfg_attr(feature = "runtime-tokio", tokio::test)]
+async fn mysql_timers_date() -> anyhow::Result<()> {
+    let mut conn = connect().await?;
+
+    // TODO: maybe use macro here? but is it OK to include `time` as test dependency?
```

Comment:
this question also holds

Reply:
I don't personally think it matters that much. It's _probably_ fine to bring it in as a test dependency, although there's no way to make it optional so it's always going to get fetched and built. @mehcode ?